tools/helixer/helixer.xml

```diff
@@ -0,0 +1,159 @@
+                <param argument="--overlap-core-length" type="integer" min="0" value="160380" label="overlap_core_length" help="Smaller values may lead to better ' 'predictions but will take longer. "></param>
+            </when>
+        </conditional>
+        <param argument="--species" type="text" value="" label="Species" help="Enter the species">
+            <sanitizer invalid_char="">
+                <valid initial="string.letters,string.digits">
+                    <add value="_" />
+                </valid>
+            </sanitizer>
+            <validator type="regex">[0-9a-zA-Z_]+</validator>
+        </param>
+        <section name="predictions_parameters" title="Predictions parameters">
+            <param argument="--batch-size" type="integer" min="0" max="100" value="8" label="batch_size" help="The batch size for the raw predictions in TensorFlow"></param>
+            <conditional name="option_overlap">
+                <param name="Option_overlaps" type="select" label="Advanced options for overlaps parameters" >
+                    <option value="yes">Show</option>
+                    <option value="no">Hide</option>
+                </param>
+                <when value="no">
+                    <param argument="--no-overlap" type="boolean" truevalue="--no-overlap" falsevalue="" checked="false" label="No overlaps" />
+                </when>
+                <when value="yes">
+                    <section name="Overlap" title="Overlap parameters">
+                        <param argument="--overlap-offset" type="integer" label="Offset of the overlap processing" help="Smaller values may lead to better predictions but will take longer.The subsequence_length should be evenly divisible by this value. (Default is subsequence_length / 2)" />
+                        <param argument="--overlap-core-length" type="integer" label="Overlap core length" help="Smaller values may lead to better predictions but will take longer. Has to be smaller than subsequence_length (Default is subsequence_length * 3 / 4)" />
+                    </section>
+                </when>
+        </conditional>
+        </section>
+        <section name="Post_processing" title="Post-processing">
+            <param argument="--window-size" type="integer" min="0" value="100" label="window size" />
+            <param argument="--min-coding-length" type="integer"  min="0" value="100" label="Minimum coding length" />
+            <param argument="--edge-threshold" type="float" min="0" max="1" value="0.1" label="Edge threshold" />
+            <param argument="--peak-threshold" type="float" min="0" max="1" value="0.8" label="Peak threshold" />
+        </section>
+        <param argument="--gff-output-path" name="output" type="data" format="gff3" label="GFF3 output"></param>
+    </inputs>
+    <outputs>
+        <data name="output" format="gff3">
+            <change_format>
+                <when input="output_format" value="fasta" format="fasta"></when>
```

Comment:
is it reall possible to get a fasta output? any way, the `output_format` param is not defined